spec/user_spec.rb

```diff
@@ -6,5 +6,10 @@
     expect(user).not_to be nil
   end
 
-  # Write a new unit test here.
-end
\ No newline at end of file
+  it 'should create a name for the user' do
+    user = User.create
+    user.name = 'Amber'
+    user.save
+    expect(user.name).to eq 'Amber'
+  end
```

Comment:
Yes. It's hard to think in these terms as a beginner:
"Imagining what WOULD have happened if I was done with my code, how do I write my test right now for what will happen when I will get my code right?"

It's a recursive thought in itself.